.github/workflows/ci.yml

```diff
@@ -75,10 +48,12 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install
-        run: make install
+        run: |
+          pip3 install --no-compile --editable .
```

Comment:
There is extreme overuse of editable in Armory (new and old) materials. Most people aren't developing Armory so we should never provide `--editable` without really good cause.

In CI, `--editable` is non-sensical unless you are testing editable